lib/app/components/atoms/custom_footer.dart

```diff
@@ -20,8 +21,10 @@ class CustomFooter extends StatelessWidget {
       ),
       child: Container(
         width: double.infinity,
-        //FIXME: 高さ指定は端末幅基準に直す
-        height: 90,
+        height: Platform.isAndroid
+            ? MediaQuery.of(context).size.height * 0.10
+            //下記はPlatform.isIOSでの高さを想定。OSの種類が増えた場合は下記を分岐させる。
+            : MediaQuery.of(context).size.height * 0.15,
```

Comment:
```suggestion
            ? Get.height * 0.10
            //下記はPlatform.isIOSでの高さを想定。OSの種類が増えた場合は下記を分岐させる。
            : Get.height * 0.15,
```
ここは私も上手いやり方が思いつかないので、一旦はこのやり方で見てみましょう。
好みですが、`Get.height`でもデバイスの高さが取れるのでこちらにしましょうか

Reply:
@king-kazu39 
本プルリクの範囲外のfollow_button.dartとhome_view.dartでもMediaQuery.of(context).sizeを使用していますがこちらも併せて修正してしまいますね。